pset-iterators.js

```diff
@@ -106,8 +117,18 @@ between 1850 and 1900. Save the data in a variable called `presidentsBetween1850
 ************/
 console.log('Problem 3:')
 
+// return all of the presidents then
+
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter(dateInOffice => {
```

Comment:
@Defy1x be careful when naming your iterator (`dateInOffice`) - remember the iterator represents the "singular" version of the elements in the array ("presidents") - a better name fro the iterator would have been `president` not `dateInOffice`

Reply:
Noted.